yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/CameraGraphicView.kt

```diff
@@ -29,40 +30,46 @@ class CameraGraphicView constructor(
 
     private var boundingBox: RectF? = null
 
+    private var imageBitmap: Bitmap? = null
+
     override fun draw(canvas: Canvas) {
         super.draw(canvas)
 
-        if (this.boundingBox == null) {
-            return
+        boundingBox?.let {
+
+            imageBitmap?.let {bitmap ->
+                canvas.drawBitmap(BlurBuilder.blur(context, bitmap), null, it, null)
+            }
```

Comment:
What do you think to draw the `imageBitmap` even if the `boundingBox` is `null`?